<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseUI</class>
 <widget class="QWidget" name="DatabaseUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatabaseUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>6</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Data Controls</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_refreshDb">
         <property name="text">
          <string>Download data from Wiki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_forceScanForRedirections">
         <property name="text">
          <string>Force Scan for Redirections</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_exportData">
         <property name="text">
          <string>Export data...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Wiki Overview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_titleFilter"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="SpreadsheetView" name="table_dbView">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title Filter (Regex):</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Session Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="textEdit_log">
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpreadsheetView</class>
   <extends>QTableView</extends>
   <header>gui/spreadsheetview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
